api/v1/region_types.go

```diff
@@ -1,8 +1,8 @@
 package v1
 
 import (
-	helmapi "github.com/fluxcd/helm-controller/api/v2beta2"
-	srcapi "github.com/fluxcd/source-controller/api/v1beta2"
+	helmapi "github.com/fluxcd/helm-controller/api/v2"
```

Comment:
I would avoid hiding api version change behind an import alias that remains the same.

```suggestion
helmapiv2 "github.com/fluxcd/helm-controller/api/v2"
```